Ruby/10001st-number.rb

```diff
@@ -0,0 +1,8 @@
+require 'prime'
+
+def prime_number
+  Prime.first (10001)
```

Comment:
in this case, the method is not handling any logic is only calling Prime.first (10001). I think is better do:

`puts "The 10001 prime number is :  #{Prime.first (10001).max}"`

Reply:
assign the 10001 to a constant this is a magic number.